docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -402,17 +402,104 @@ feasible based on the data present, it is also a reasonable approach.
+    - *Policy advocacy*: Research informed by CRT should not only aim to understand and document disparities but also to drive change. This involves advocating for policies that address the root causes of racial disparities in health, such as economic inequality, housing, education, and access to healthcare. Researchers can play a role in informing policy, engaging in public discourse, and supporting community-led initiatives.
+    - *Reflective practice*:  Engage in ongoing reflection about your own positions, biases, and the power dynamics in your work. This includes being open to critique, actively seeking diverse perspectives, and being committed to anti-racist practices both personally and professionally.
+  
+
+4. **Cite the experts, particularly scholars of color whose work forms the basis of the field’s knowledge on racism and its effects** 
+[Boyd_2020]_.
+
+
+Reporting results accurately
+----------------------------
+
 .. todo:: 
 
-    In this section we'll discuss the question "How do I frame my communication about race and ethnicity without stigmatizing/othering?" and how a health 
-    metrics researcher would answer this question and use that answer to inform their communications and language.
+    Use recommendations from sex/gender guide to fill out this section. This section should include 
+    a. Outline goals when we present our race/ethnicity-related findings
+    b. General strategy (ie explain fully in standalone parts of paper, otherwise use shorthand) - Recs for detailed description of methods with examples and recs for abbreviated description with examples 
 
 4.0 References
 ++++++++++++++
 
+.. [AMA_2021]
+    American Medical Assiociation. Advancing Health Equity: A Guide to Language, Narrative and Concepts. 2021. https://www.ama-assn.org/system/files/ama-aamc-equity-guide.pdf
+
+.. [Boyd_2020]
+    Boyd RW, Lindo EG, Weeks LD, McLemore MR. On Racism: A New Standard for Publishing on Racial Health Inequities. Health Affairs Blog. 2020 Jul. doi: 10.1377/hblog20200630.939347
+
+.. [Calanan_2023]
+    Calanan RM, Bonds ME, Bedrosian SR, Laird SK, Satter D, Penman-Aguilar A. CDC’s Guiding Principles to Promote an Equity-Centered Approach to Public Health Communication. Prev Chronic Dis 2023;20:230061. DOI: http://dx.doi.org/10.5888/pcd20.230061
+
+.. [Cerdena_2020]
+    Cerdena JP, Plaisime MV, Tsai J. The Lancet. 2020 Oct; 396(10257):1125-1128. doi: https://doi.org/10.1016/S0140-6736(20)32076-6
+
+.. [Ford_Airhihenbuwa_2010]
+    Ford CL, Airhihenbuwa CO. Critical Race Theory, Race Equity, and Public Health: Toward Antiracism Praxis
```

Comment:
Maybe add a link to the article here, and/or the DOI:

PubMed Central: https://pmc.ncbi.nlm.nih.gov/articles/PMC2837428/

DOI: https://doi.org/10.2105/AJPH.2009.171058

Reply:
Oh, and also a more complete citation:

> Ford CL, Airhihenbuwa CO. Critical Race Theory, race equity, and public health: toward antiracism praxis. Am J Public Health. 2010 Apr 1;100 Suppl 1(Suppl 1):S30-5. doi: 10.2105/AJPH.2009.171058. Epub 2010 Feb 10. PMID: 20147679; PMCID: PMC2837428.

And then you can make the DOI and the PMID into hyperlinks using the links above.